### PR TITLE
[plugin] repeatedly check webview for readiness

### LINKED ISSUE #5033: [plugin] webview only tries once to send message to iframe
This is a follow up to https://github.com/theia-ide/theia/issues/4267.

When a plug-in has a webview, we defer posting messages to the the webview's iframe until it is ready. Unfortuntely, the current impl checks for readiness immediately and once after 100ms. If initialization takes longer that 100ms the message gets lost.

### PR DESCRIPTION
**Pull request by JanKoehnlein**
Fixes #5033

### CODE REVIEW

File: packages/plugin-ext/src/main/browser/webview/webview.ts [hunk 1]
```diff
@@ -266,21 +266,19 @@ export class WebviewWidget extends BaseWidget {
     waitReceiveMessage(object: WebviewWidget, resolve: any) {
         if (object.readyToReceiveMessage) {
             resolve(true);
+        } else {
+            setTimeout(this.waitReceiveMessage, 100, object, resolve);
```
Comment (marechal-p) [inline packages/plugin-ext/src/main/browser/webview/webview.ts]: Don't we need to bind the function here?

Comment (marechal-p) [reply packages/plugin-ext/src/main/browser/webview/webview.ts]: [Apparently not](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this#As_an_object_method), my bad.

**Comment by benoitf** [comment]
I'm on pto this week so review are longer than usual. For this kind of this PR/ hotfix, I'm fine if it's merged without my approval

**Comment by akosyakov** [comment]
@JanKoehnlein I've tried your example but it does not show anything. Maybe we have a look together today?

**Pull request merged**